NAMESPACE

```diff
@@ -1,18 +1,22 @@
 # Generated by roxygen2: do not edit by hand
 
 S3method("[",epiflows)
+S3method(as.SpatialLinesDataFrame,epiflows)
 S3method(epiflows,data.frame)
 S3method(epiflows,integer)
 S3method(epiflows,numeric)
+S3method(get_coords,epiflows)
 S3method(get_flows,epiflows)
 S3method(get_locations,epiflows)
 S3method(get_vars,epiflows)
 S3method(plot,epiflows)
 S3method(print,epiflows)
 export(add_coordinates)
+export(as.SpatialLinesDataFrame)
 export(epiflows)
 export(estimate_risk_spread)
 export(get_codes)
+export(get_coords)
```

Comment:
change to `get_coordinates()` for name consistency with `add_coordinates()`

Reply:
Done.